include/tudat/io/readVariousPdsFiles.h

```diff
@@ -0,0 +1,46 @@
+/*    Copyright (c) 2010-2023, Delft University of Technology
+ *    All rigths reserved
+ *
+ *    This file is part of the Tudat. Redistribution and use in source and
+ *    binary forms, with or without modification, are permitted exclusively
+ *    under the terms of the Modified BSD license. You should have received
+ *    a copy of the license with this file. If not, please or visit:
+ *    http://tudat.tudelft.nl/LICENSE.
+ *
+ *    References: 820-013, TRK-2-18 Tracking System Interfaces Orbit Data File Interface, Revision E, 2008, JPL/DSN
+ *
+ */
+
+#ifndef TUDAT_READ_VARIOUS_PDS_FILES_H
+#define TUDAT_READ_VARIOUS_PDS_FILES_H
+
+#include <iostream>
+#include <fstream>
+#include <bitset>
+#include <cmath>
+#include <vector>
+#include <map>
+
+#include "tudat/astro/basic_astro/timeConversions.h"
+#include "tudat/io/basicInputOutput.h"
+#include "tudat/astro/earth_orientation/terrestrialTimeScaleConverter.h"
+
+namespace tudat
+{
+namespace input_output
+{
+
+
+double computeTimeTdbFromUtc( double timeUtc );
```

Comment:
This seems like an odd place for this function. Is it to convert time stamps of observations from UTC to TDB? Shouldn't the station location then also matter?

Reply:
oh my bad, this should not be part of the pull request indeed. I'd exclude the readVariousPdsFiles.h/.cpp for now (still drafty file readers). sorry about this

Reply:
something we should discuss though, because I was unsure about how to perform this time conversion without information on the station (there is no station actually, since this is used for Grail's mass files)